### ISSUE TITLE
Sitemap Indexes within another index?

### ISSUE THREAD

**Issue by yovanoff** [evermade/yoast-seo-multisite-subdirectory-sitemaps#1]
Hello, and thank you for this plugin!

I was wondering if you've ever seen any issues with it. As far as I can tell, this adds links to other sitemap _indexes_ to the main sitemap index provided by Yoast.

But in my understanding of the [protocol](https://www.sitemaps.org/protocol.html) and Google's [description](https://developers.google.com/search/docs/crawling-indexing/sitemaps/large-sitemaps), a sitemap index can only include links to other XML _sitemaps_, it cannot link to other XML _sitemap indexes_.

Perhaps I'm misunderstanding the protocol or the plugin? Have you ever run in to any issues with this submitting sitemaps correctly?

Thank you!

**Comment by bronglil** [comment]
You’re correct; according to the [sitemaps.org protocol](https://www.sitemaps.org/protocol.html), a sitemap index should ideally link only to individual XML sitemaps rather than to other sitemap indexes. Google’s documentation on [large sitemaps](https://developers.google.com/search/docs/crawling-indexing/sitemaps/large-sitemaps) also supports this by emphasizing that each index file is expected to link directly to regular XML sitemaps for optimal indexing.

If your current setup results in nested sitemap indexes, I recommend restructuring them so that only individual XML sitemaps are listed within the main index. This approach will align with the protocol’s specifications and could prevent any potential indexing issues.

If you're experiencing warnings in Google Search Console related to the sitemap structure, adjusting it as described above should improve compliance. For additional configuration help, you might also reach out to Yoast support, as they may offer solutions that align better with these guidelines.
